docs/compilers/CSharp/Runtime Async Design.md

```diff
@@ -27,20 +27,32 @@ We use the following helper APIs to indicate suspension points to the runtime, i
 ```cs
 namespace System.Runtime.CompilerServices;
 
-// These methods are used to await things that cannot use runtime async signature form
 // TODO: Clarify which of these should be preferred? Should we always emit the `Unsafe` version when awaiting something that implements `ICriticalNotifyCompletion`?
 namespace System.Runtime.CompilerServices;
 
 public static class RuntimeHelpers
 {
-    [RuntimeAsyncMethod]
+    // These methods are used to await things that cannot use the Await helpers below
+    [MethodImpl(MethodImplOptions.Async)]
     public static Task AwaitAwaiterFromRuntimeAsync<TAwaiter>(TAwaiter awaiter) where TAwaiter : INotifyCompletion;
-    [RuntimeAsyncMethod]
+    [MethodImpl(MethodImplOptions.Async)]
     public static Task UnsafeAwaitAwaiterFromRuntimeAsync<TAwaiter>(TAwaiter awaiter) where TAwaiter : ICriticalNotifyCompletion;
```

Comment:
I think it is most natural for these to be void returning, similar to the intrinsic Await helpers below.

Reply:
I missed that Andy had updated these signatures as well in his PR, I'll update this side.